### DISCUSSION TITLE
May I know why webpack-dev-server@5 needs a version of node 18 at least?

### DISCUSSION THREAD

**Discussion by Indomi** [webpack/webpack#18618] [Q&A]
Hi guys, I am developing an old program with node version 16. When I try to upgrade to use webpack5, I notice that the node version dependency of webpack-dev-server@5 is 18. May I know why? Are there any features or breaking changes that we must have this limit?
Hope to get answers. Thanks a lot.

**Comment by alexander-akait** [comment]
Because Node.js@16 is EOL
